imagetest/test_suites/image_validation/cloud_test.go

```diff
@@ -0,0 +1,60 @@
+package imagevalidation
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+// TestNTPService Verify that ntp package exist and configuration is correct.
+func TestNTPService(t *testing.T) {
```

Comment:
I test this on centos, rhel, ubuntu and debian. There is no /etc/ntp.conf and ntpd service. So I doubt this is a validate test case.

Reply:
```
=== RUN   TestNTPService
    cloud_test.go:21: open /etc/ntp.conf: no such file or directory
--- FAIL: TestNTPService (0.00s)
FAIL```

Reply:
Most things are moving to chrony- so we need to look for both now. The configuration for chrony is /etc/chrony/chrony.conf but differs. Take a look at Debian 10 for an example.

Reply:
sles12 has ntp.conf if you want to use that to test

Reply:
Modify the test for both ntpd and chronyd service according distros